docs/clustering/multi-node-setup.rst

```diff
@@ -319,10 +319,14 @@ the quorum size using the `discovery.zen.minimum_master_nodes`_ setting.
 
 .. NOTE::
 
-    For a three-node cluster, CrateDB will set the quorum size to three.
-    Consequentially, you must declare all nodes to be master-eligible. Consult
-    the `quorum guide`_ for detailed information about quorum size
-    considerations.
+    CrateDB will set the quorum size to ``FLOOR(n / 2) + 1``, so that is the 
+    minimum number of master-eligible nodes you should have.
+    For high availability purposes you want to have one additional
+    master-eligible node on top of the minimum required for quorum for each
+    node loss you want to tolerate. A 3-nodes cluster for instance with
```

Comment:
For consistency, further down in the file the spelling "three-node cluster" is used.